tools-public/toolpad/components/PieChart.tsx

```diff
@@ -16,29 +16,26 @@ const COLORS = [
 ];
 
 export interface PieChartProps {
-  data: object[];
+  data: any[];
 }
 
 function PieChartExport({ data }: PieChartProps) {
   return (
-    <PieChart width={300} height={300}>
-      <Pie
-        data={data}
-        cx={150}
-        cy={150}
-        innerRadius={0}
-        outerRadius={80}
-        fill="#8884d8"
-        dataKey="value"
-      >
-        {data.map((entry, index) => (
-          <Cell key={`cell-${index}`} fill={COLORS[index % COLORS.length]} />
-        ))}
-      </Pie>
-      <Tooltip
-        formatter={(value) => Intl.NumberFormat('en', { notation: 'compact' }).format(value)}
-      />
-    </PieChart>
+    <PieChart
+      series={[
+        {
+          data: data.map((entry, index) => ({
+            id: index,
+            label: entry.name,
+            color: COLORS[index % COLORS.length],
+            value: entry.value,
+          })),
+          valueFormatter: ({ value }) => Intl.NumberFormat('en', { notation: 'compact' }).format(value),
```

Comment:
I was expecting 
```suggestion
          valueFormatter: (value) => Intl.NumberFormat('en', { notation: 'compact' }).format(value),
```

per this demo: https://next.mui.com/x/react-charts/tooltip/#formatting, strange.